dts/bindings/arm/nxp,hci-ble.yaml

```diff
@@ -0,0 +1,12 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: HCI interrupt info
```

Comment:
since the binding  name is generically "hci-ble" the description should match this or the name should match the description

Reply:
Corrected,Thanks.